easybuild/easyblocks/v/vep.py

```diff
@@ -156,9 +156,7 @@ def make_module_req_guess(self):
             perl_ver = get_software_version('Perl')
             perl_libpath.extend([os.path.join('lib', 'perl' + perl_majver, 'site_perl', perl_ver)])
 
-        guesses = super(EB_VEP, self).make_module_req_guess()
-        guesses = {
-            'PATH': '',
-            'PERL%sLIB' % perl_majver: perl_libpath,
-        }
-        return guesses
+        self.module_load_environment.PATH = ''
+        setattr(self.module_load_environment, 'PERL%sLIB' % perl_majver, perl_libpath)
```

Comment:
:heart: f-strings :heart:
```suggestion
        setattr(self.module_load_environment, f'PERL{perl_majver}LIB', perl_libpath)
```